scripts/workflows/spdk_jobfile.fio

```diff
@@ -0,0 +1,20 @@
+[global]
+ioengine=spdk_bdev
+spdk_json_conf=/workspace/scripts/workflows/bdev.json
+thread=1
+group_reporting=1
+direct=1
+verify=0
+norandommap=1
+time_based=1
+ramp_time=60s
+runtime=600s
+bs=4k
+numjobs=1
+loops=3
+rw=${RW}
+iodepth=${IOD}
+rwmixread=${MIX}
+
+[test]
+filename=/dev/nvme0n1
```

Comment:
it'd be great if we can expose this as an environment variable, so that it can be specified in the `env` attribute. can be made a global one:

```yaml
options:
  env:
    DEVICE: /dev/nvme0n1

steps:
....
```

and then we use `${DEVICE}` in the `.fio` files